validator.Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM golang:1.15 as builder
+
+ARG VERSION=latest
+ARG COMPONENT="kubevirt-template-validator"
+ARG BRANCH=master
+ARG REVISION=master
+
+WORKDIR /workspace
+
+# Copy the Go Modules manifests and vendor directory
+COPY go.mod go.mod
+COPY go.sum go.sum
+COPY vendor/ vendor/
+
+# Copy the go source
+COPY internal/ internal/
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -ldflags="-X 'kubevirt.io/ssp-operator/internal/template-validator/version.COMPONENT=$COMPONENT'\
+-X 'kubevirt.io/ssp-operator/internal/template-validator/version.VERSION=$VERSION'\
+-X 'kubevirt.io/ssp-operator/internal/template-validator/version.BRANCH=$BRANCH'\
+-X 'kubevirt.io/ssp-operator/internal/template-validator/version.REVISION=$REVISION'" -o internal/template-validator/_out/kubevirt-template-validator internal/template-validator/main.go
```

Comment:
The output can be in the working directory:
```
-o kubevirt-template-validator
```

Using the long path only makes the dockerfile more complex.